src/dyn/qtag/CMakeLists.txt

```diff
@@ -25,5 +25,7 @@ endif()
 #
 TARGET_LINK_LIBRARIES(qtag      linalg_stat ${ext_libs})
 TARGET_LINK_LIBRARIES(qtag_stat linalg_stat ${ext_libs})
+TARGET_LINK_LIBRARIES(qtag      nhamiltonian      ${ext_libs})

```

Comment:
just add "nhamiltonian_stat" (btw, I'm linking vs the static libraries, so it would be nhamiltonian_stat not nhamiltonian) in the list with "linalg_stat"  - why to separate into several lines?